MainWindow.h

```diff
@@ -27,19 +26,35 @@ class MainWindow;
 class MainWindow : public QMainWindow {
   Q_OBJECT
 
+ private:

```

Comment:
This move confused me for a second. As a style rule, we (primarily Google) put the private block at the bottom of the class (so that things we want people to know about are on top).